common/v2/components/LinkOut.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Typography } from 'v2/components';
+import { COLORS } from 'v2/theme';
+
+import LinkOutIcon from 'assets/images/link-out.svg';
+
+const { BRIGHT_SKY_BLUE } = COLORS;
+
+const LinkOutWrapper = styled.a`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  p {
+    margin: 0;
+  }
+`;
+
+const OutIcon = styled.img`
+  color: ${BRIGHT_SKY_BLUE};
+  width: 18px;
+  height: 18px;
+  margin-left: 10px;
+`;
+
+interface Props {
+  text: string;
+  link: string;
+  truncate?(text: string): string;
+}
+
+const LinkOut = ({ text, link, truncate }: Props) => {
+  const content = truncate ? truncate(text) : text;
+  return (
+    <LinkOutWrapper href={link} target="_blank">
```

Comment:
Every `target="_blank">` should be accompanied with `noreferrerr`  & `nooopener`
https://developers.google.com/web/tools/lighthouse/audits/noopener

I'm for merging all our different link components into a single one.
We can continue discussing on slack

Reply:
Thanks for the info, gonna add this asap !

Let's discuss ont this.